app/views/vacancies/_show.html.slim

```diff
@@ -19,17 +19,10 @@
                                              similar_jobs: @similar_jobs,
                                              invented_job_alert_search_criteria: @invented_job_alert_search_criteria
 
-.govuk-grid-row class="#{current_variant_container_name(:similar_jobs_position)} govuk-!-margin-top-6"
+.govuk-grid-row class="govuk-!-margin-top-6"
   .govuk-grid-column-one-third
     = render "vacancies/listing/key_dates_sidebar", vacancy: @vacancy, similar_jobs: @similar_jobs
 
-    - if current_variant?(:similar_jobs_position, :left)
-      = render "vacancies/listing/similar_jobs", vacancy: @vacancy,
-                                                 similar_jobs: @similar_jobs,
-                                                 invented_job_alert_search_criteria: @invented_job_alert_search_criteria,
-                                                 linked_locations: linked_locations(@vacancy),
-                                                 column_one_half: false
```

Comment:
the logic for column_one_half can also be removed in the partial and from the other call to render the partial